src/keyswitch_state.cpp

```diff
@@ -11,7 +11,7 @@ boolean key_was_pressed (byte keyState) {
 }
 
 boolean key_was_pressed (uint8_t currentState, uint8_t previousState) {
-    if (previousState) {
+    if (previousState & B00000001) {
```

Comment:
Any reason this (and the ones below) are not using the `INJECTED` define?  That would seem clearer to me.